README.md

```diff
@@ -22,7 +22,7 @@ For more details, see our [FAQ](https://www.cockroachlabs.com/docs/frequently-as
 
 ## Status
 
-CockroachDB is currently in alpha. See our
+CockroachDB is currently in beta. See our
```

Comment:
Oops... I should have waited a few more seconds... I see the merge now... and the updated README on github - "CockroachDB is currently in beta". Excellent!